File: contest/LeetCode_Weekly_Contest_193/Running_Sum_of_1d_Array.cpp
class Solution {
public:
    vector<int> runningSum(vector<int>& nums) {
        vector<int> ret(nums.size());
        ret[0] = nums[0];
        for (int i = 1; i < nums.size(); ++i)
            ret[i] = ret[i-1] + nums[i];
        return ret;
    }
};

File: activity/20200805/EricDuy/Running-sum-of-1d-array.java
class Solution {
    public int[] runningSum(int[] nums) {
        if(nums.length==1) return nums;
        nums[1] = nums[0] + nums[1];
        for(int i = 2; i < nums.length; i++){
            nums[i] = nums[i] + nums[i-1];
        }
        return nums;
    }
}
